tutorials/dapt-curation/code/configs/text_semantic_dedupe_config.yaml

```diff
@@ -0,0 +1,32 @@
+# Configuration file for semdantic dedup
+cache_dir: "workspace/semdedup_cache/text"
+num_files: 16
+id_col_name: "id"
+id_col_type: "str"
+input_column: "text"
+
+# Embeddings configuration
+embeddings_save_loc: "embeddings"
+embedding_model_name_or_path: "sentence-transformers/all-MiniLM-L6-v2"
+embedding_batch_size: 128
+embedding_max_mem_gb: 20
```

Comment:
https://github.com/NVIDIA/NeMo-Curator/pull/327 removed `id_col_name`, `id_col_type`, `input_column`, and `embedding_max_mem_gb` from the configuration file. It also renamed `id_col_name` to `id_column` and `id_col_type` to `id_column_type`. Please update. Thanks!